exe/Server/DB.hs

```diff
@@ -25,11 +25,14 @@ getSinceDB conn name id
 -- | Get all txs in all chains. Useful after a server restart.
 getAllDB :: Connection -> IO [(Text, [Value])]
 getAllDB conn = do
-    res :: [(Int, Text, Value)] <- query_ conn "SELECT id, chain, expr FROM txs"
+    -- The @ORDER BY@ clauses are important, because
+    -- * 'groupBy' only groups adjacent items in a list, and
```

Comment:
Maybe use a `GROUP BY`?

Reply:
As far as I understand it `GROUP BY` in SQL only makes sense for aggregates. How would you use it to make the groups here?

Reply:
Ah yeah I got a bit to used to neo4j. You can use `array_agg` but probably not worth it.